pynos/versions/base/interface.py

```diff
@@ -3218,11 +3236,20 @@ def vrrpe_vmac(self, **kwargs):
             callback (function): A function executed upon completion
             of the  method.  The only parameter passed to `callback`
             will be the ``ElementTree`` `config`.
+
         Returns:
             Return value of `callback`.
         Raises:
             KeyError: if `int_type`, `name`, `vrid`, or `vmac` is not passed.
             ValueError: if `int_type`, `name`, `vrid`, or `vmac` is invalid.
+
```

Comment:
I still see duplicate Returns:
              Return value of `callback`.
          Raises:
              KeyError: if `int_type`, `name`, `vrid`, or `vmac` is not passed.
              ValueError: if `int_type`, `name`, `vrid`, or `vmac` is invalid.